app/decorators/json_schema/attributes_resource_representation_decorator.rb

```diff
@@ -54,7 +54,7 @@ def item_property
     def primitive_property
       {
         type: resource_attribute.primitive_type,
-        format: format,
+        format: if format == "datetime" then "date-time" else format end,
```

Comment:
Ternary is a bit more idiomatic in Ruby :  format: format == "datetime" ? "date-time" : format